src/components/Card/Card.tsx

```diff
@@ -0,0 +1,169 @@
+
+        const bodyClasses: string = mergeClasses([styles.body, bodyClassNames]);
+
+        const footerClasses: string = mergeClasses([
+            styles.footer,
+            footerClassNames,
+        ]);
+
+        return (
+            <div {...rest} className={cardClasses} ref={ref} style={style}>
+                <div className={headerClasses}>
+                    <div className={styles.mainHeader}>
+                        <Icon
+                            path={icon}
+                            classNames={styles.icon}
+                            size={IconSize.Large}
+                        />
+                        <div className={styles.title}>
+                            {header}
+                            <div className={styles.subHeader}>
+                                {subHeaderProps &&
+                                    subHeaderProps.map((item, idx) => {
+                                        return (
+                                            <div>
+                                                {idx ===
+                                                    subHeaderProps.length -
+                                                        1 && <>&nbsp;&nbsp;</>}
+                                                {item}
+                                                {idx <
+                                                    subHeaderProps.length -
+                                                        1 && (
+                                                    <>&nbsp;&nbsp;&bull;</>
+                                                )}
+                                            </div>
+                                        );
+                                    })}
+                            </div>
+                        </div>
+                    </div>
+                    <div className={styles.buttonIcon}>
+                        {headerButtonProps && (
+                            <TwoStateButton
+                                classNames={styles.mainHeaderButton}
+                                shape={ButtonShape.Round}
+                                iconOneProps={{
+                                    path: headerIcon,
+                                    ariaHidden: true,
+                                    classNames: 'my-two-state-btn-icon-one',
+                                    id: 'myTwoStateButtonIconOne',
+                                    rotate: 0,
+                                    spin: false,
+                                    vertical: false,
+                                    'data-test-id':
+                                        'myTwoStateButtonIconOneTestId',
+                                }}
+                                {...headerButtonProps}
+                            />
+                        )}
+                    </div>
+                </div>
+                <div className={bodyClasses}>
+                    {bodyListOneProps && (
```

Comment:
There should be a two cases supported here:

A render that supports fully customized content
The default where the Card provides the content to be populated.

The default cases will need to be a best effort at balancing opinionated layout and scalability.